pkg/pubsub/pubsub.go

```diff
@@ -58,29 +57,17 @@ func (ps *PubSub) String() string {
 }
 
 func (ps *PubSub) GetIDName() string {
```

Comment:
these getters need to be removed

Reply:
line 35 `Version` should be removed.

Reply:
line 28-30 example needs to update with new O-RAN names

Reply:
@jzding In this PR, I deliberately did not touch external APIs.